Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -328,7 +328,6 @@ open class CarPlayMapViewController: UIViewController {
         setupStyleManager()
         setupSpeedLimitView()
         setupWayNameView()
-        navigationMapView.navigationCamera.follow()
```

Comment:
`NavigationMapView` already switches to the following mode upon creation, there is no need to call this method here.